apps/rehash.c

```diff
@@ -51,6 +51,26 @@
 # endif
 # define MAX_COLLISIONS  256
 
+# if defined(OPENSSL_SYS_VXWORKS)
+/* 
+ *   VxWorks has no symbolic links 
+ */
+
+#  define lstat(path, buf) stat(path, buf)
+
+int symlink(const char *target, const char *linkpath)
+{
+    errno = EACCES;
```

Comment:
Shouldn't this be [ENOSYS instead of EACCESS](http://man7.org/linux/man-pages/man3/errno.3.html)?

Reply:
And since you started: shouldn't this errno be set consistently for all other functions which are not implemented (including `setpgid()`, `fork()`)?